.github/workflows/build.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       run: ./build.ps1
 
     - name: Upload Coverage Reports
-      if: always()
+      if: always() && github.repository == 'App-vNext/Polly'
```

Comment:
What's the benefit of not uploading the artifacts if someone decides to run this workflow in their fork?

Reply:
I would consider it to be more efficient in terms of runtime and space used. Furthermore, especially for the codedov uploads which will fail for sure because they don't have access to the secrets.

Reply:
This step doesn't upload anything to codecov - they're being uploaded to the artifacts of the workflow run itself.